GoRest.Api.Tests/Comments/Create/CreateCommentsNegative.cs

```diff
@@ -0,0 +1,118 @@
+﻿using API_Tests.Asserts;
+using API_Tests.Helpers;
+using FluentAssertions;
+using GoRest.Api.Client.Client;
+using GoRest.Api.Client.Client.Builder;
+using GoRest.Api.Client.Client.Extentions;
+using GoRest.Api.Client.Client.Interfaces.Controllers;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace API_Tests.Posts.Create.CreateCommentNegativeTests
+{
+    [Parallelizable]
+    [TestFixture]
+    public class CreateCommentNegativeTests
+    {
+        [Test]
+        public async Task VerifyCommentIsNotCreateWhenEmptyFields()
+        {
+            // Arrange
+            (string userId, string postId) createdPost = await new CreateEntities().CreatePost();
+            var commentModelEmptyFields = new CreateCommentBuilder().With(x =>
+            {
+                x.Name = "";
+                x.Email = "";
+                x.Body = "";
+            }
+              ).Build();
+
+            //Act
+            var responseCreateComment = await GoRestClient.For<ICommentsApi>().CreateCommentNegative(commentModelEmptyFields, createdPost.postId);
+
+            // Assert
+            CommentsAsserts.VerifyCommentFieldsCannotBeBlank(responseCreateComment);
+        }
+
+        [Test]
+        public async Task VerifyCommentIsNotCreateWhenFieldsExceedLength()
+        {
+            // Arrange
+            (string userId, string postId) createdPost = await new CreateEntities().CreatePost();
+            var commentModelExceedLentghFields = new CreateCommentBuilder().With(x =>
+            {
+                x.Name = "QA_201_characters_entered_negative_tests ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exer";
```

Comment:
let`s generate random value with specific length instead of hardcoding

Reply:
added method  for random string Builder
 new RandomStringBuilder().GenerateRandomStringOfSpecifiedLength(201);